drivers/hmis/app/models/hmis/hud/processors/current_living_situation_processor.rb

```diff
@@ -22,5 +22,15 @@ def schema
     def dependent_destroyable?
       true
     end
+
+    def assign_metadata
+      super
+
+      current_living_situation = @processor.send(factory_name, create: false)
+      return unless current_living_situation.verified_by_project_id
```

Comment:
Hmm. I'm not sure how this should behave if `verified_by_project_id` is empty. On the one hand, I would expect that if I clear that value and save, that the VerifiedBy field is cleared.

On the other hand, if we have imported records with CurrentLivingSituation.VerifiedBy, I would expect that value to remain even when updating the form.
I might be forgetting here, did we discuss the desired behavior for migrated-in records with VerifiedBy values?

Reply:
If we pivot to storing one of the Project IDs in this field instead of name, could we?
1. Attempt to match a project and show the name if we can
2. Show the actual value if we can't

Reply:
Are you suggesting storing ProjectID in the HUD `VerifiedBy` column? We did discuss that option. I'm not sure I quite understand. I can provide some more context if that's what you're asking about though

Reply:
Yeah, my suggestion would be to store the `ProjectID` in `VerifiedBy`.  As long as we can provide a drop-down of projects to fill the field, I don't think HUD will ever verify if the field contains the name or ID.

Reply:
Okay, I provided some context on https://github.com/open-path/Green-River/issues/6272#issuecomment-2266013278 and suggested reopening this for discussion at the hmis check-in call

Reply:
it seems like something like this could work? 

```ruby
# if this was a normal mutation, could logic would be:
if input.verified_by_project_id
  # allow updating verified_by
  cls.verified_by = find_project_name(input.verified_by_project_id)
else if cls.verified_by_project_id
  # cls was set previously and is now cleared explicitly
  current_living_situation.verified_by = nil
else
  # current_living_situation.verified_by is preserved
end
```
So an imported cls.verified_by value would be unchanged since cls.verified_by_project_id would be null. But a user could still change the value (and once changed could then clear it as well)